package pl.szymanski.sharelibrary.exceptions;

public class ExceptionMessages {

    public static final String BOOK_DOES_NOT_EXIST_EXCEPTION_FORMAT = "Book with id %s does not exist";
    public static final String AUTHOR_DOES_NOT_EXIST_EXCEPTION_FORMAT = "Author %s does not exist";
    public static final String EMAIL_ALREADY_EXISTS_EXCEPTION_FORMAT = "Email %s already exists";
    public static final String USERNAME_ALREADY_EXISTS_EXCEPTION_FORMAT = "Username %s already exists";
    public static final String INVALID_EMAIL_ADDRESS_EXCEPTION_FORMAT = "Email %s is invalid";
    public static final String USER_NOT_FOUND_BY_EMAIL_OR_USERNAME_EXCEPTION_FORMAT = "User with email/username %s not found";
    public static final String USER_NOT_FOUND_BY_ID_EXCEPTION_FORMAT = "User with id %s not found";
    public static final String INVALID_USERNAME_EMAIL_OR_PASSWORD = "Invalid username, email or password";
    public static final String UPLOAD_IMAGE_EXCEPTION = "There is a problem Uploading Your Image";

    public static final String UNKNOWN_ERROR_OCCURRED = "unknown error occurred";

    public static final String COVER_WITH_ID_DOES_NOT_EXISTS = "Cover with id %s does not exist";
    public static final String EXCHANGE_DOES_NOT_EXIST = "Exchange with id %s does not exist";
}
